dataland-frontend/src/components/resources/frameworkDataSearch/euTaxonomy/EuTaxonomyForNonFinancialsPanel.vue

```diff
@@ -34,7 +34,6 @@ import { type Field } from "@/utils/GenericFrameworkTypes";
 import { euTaxonomyForNonFinancialsModalColumnHeaders } from "@/components/resources/frameworkDataSearch/euTaxonomy/EuTaxonomyForNonFinancialsModalColumnHeaders";
```

Comment:
On the view page, the total revenue is split into two fields: Value and Currency. But for all other fields with currency, they are displayed as one "[Value] [Currency]" (Which is IMO more desirable). Nevertheless, I think we should unify this ? (Up for discussion).

Reply:
Yes, i agree. But as far as i remember @emanueldfine and @SiegfriedSobkowiak please correct me if iam mistaken, we decided to go with this separate approach to avoid major refacotring at this point of time.

Reply:
Yes we are aware of the issue but currently it is the best solution to leave it like this. We have a story in the backlog to consolidate this.